package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "hermione",
-  "version": "8.7.2",
+  "name": "testplane",
+  "version": "0.0.1",
```

Comment:
first ready version will be marked as 0.1.0

Reply:
Don't forget to change to 0.1.0 in the file

Reply:
It will be made automatically. With `npm run release`